### DISCUSSION TITLE
inverse order

### DISCUSSION THREAD

**Discussion by flaviocaramit** [maoserr/epublifier#381] [Ideas]
can you add to simplify the order where things appear. Like in a site im using the last chapter is the most recent and it would be easier to just have a button that change the order bc doing this manually can be pretty exhausting

**Comment by maoserr** [comment]
yea, I'll add a sorting by column header

**Reply by maoserr** [reply]
although you could export csv/re-import after sorting as a work around

**Reply by flaviocaramit** [reply]
i will try this csv thing, thx for the hint

**Comment by maoserr** [comment]
done in next release
